unified-codes/.env

```diff
@@ -1,14 +1,26 @@
-# Default values
-## Storybook doesn't get .local.env or sub-dir .env files loaded for it. 
-## For storybook usages, enter them here.
-## For local dev, create entries in ./apps/web/.local.env
-## For production entries - these are committed and are in ./apps/web/.env
-
-NX_AUTHENTICATION_URL=http://localhost:9990/auth/realms/unified-codes
-NX_LEGACY_API_PORT=3000
-NX_DATA_SERVICE=http://localhost:4000/graphql
-NX_DATA_SERVICE_PORT=4000
-NX_DGRAPH_ENDPOINT=http://localhost:8080
-NX_CLIENT_SECRET=asecretthing
-NX_CLIENT_ID=unified-codes-data
-NX_RXNAV_ENDPOINT=https://rxnav.nlm.nih.gov/REST
\ No newline at end of file
+## For local development and storybook usages, rename this file as .env.
```

Comment:
Yes 😉.

Basically, this file should be `.env`, I don't know why I left the rename there (I think initially I named it `.dev.env` or something!). Will remove!

Reply:
So to clarify, this file is always the last env file read, which means that if no other overrides are defined, it will fall back on "debug" mode. If the `.dist.env` file is renamed to be `.local.env`, it will be read first, causing these variables to be ignored.

Yes, no, maybe?

Reply:
sure - I usually have production values in the fallback, and override with local values. Doesn't matter either way really.